tox.ini

```diff
@@ -32,6 +32,7 @@ deps =
     -r requirements-dev.txt
 commands =
     python -m pytest {posargs}
+    python -m pytest --nbmake  notebooks/simple-graph-plot-2022.ipynb notebooks/Agent-Close-Following.ipynb
```

Comment:
I see running the notebooks as a form of test.
However we could restrict it to just 3.10, say.

Reply:
I understand but we also have the `[testenv:notebooks]` step (which we don't run in the checks atm though). Wouldn't this be part of that?

Reply:
Could we enable the notebooks env in checks.yml now?  (I tried to comment out the `if false` but I get the personal access token error when I try to commit)
I have `tox -e notebooks` working with `nbmake`, with all the notebooks from the previous `run_all_notebooks.py` and `notebook-list`.  One of them needs `redis` - `test-service.ipynb` - this runs an evaluator process in the background and a client in the foreground.  It's a bit contrived / "heath robinson" but I gather @manuschn Manuel would like us to do some evaluator work so it may be worthwhile.  If it's too tricky for now then we can remove this notebook from the list in `tox.ini`.

Reply:
#64 enables the `notebooks` job in `checks.yml`; I've removed the `pytest --nbmake` from the tox.ini `testenv:py*`,
I think this resolves this thread.